shopify/session.py

```diff
@@ -10,6 +10,7 @@
 import re
 from contextlib import contextmanager
 from six.moves import urllib
+from shopify.api_access import ApiAccess
```

Comment:
I think all that's needed here is the following:

```suggestion
from shopify import ApiAccess
```

The base shopify `__init__.py` file specifies `from api_access import *`.

Reply:
It fails locally as it fails to find the module.

Reply:
Does the pytest runner fail?

Reply:
Oh it might be `from .api_access import ApiAccess`. What does the failure say specifically?

Reply:
```
from shopify import ApiAccess
E   ImportError: cannot import name ApiAccess
```

Reply:
`from .api_access import ApiAccess` works but idk if it might be better to be more explicit and keep it as `from shopify.api_access import ApiAccess`